models/discordactions.js

```diff
@@ -246,7 +246,7 @@ const enrichGroupDataWithMembershipInfo = async (discordId, groups = []) => {
     const subscribedGroupIds = findSubscribedGroupIds(discordId, groupsToUserMappings);
 
     return groups.map((group) => {
-      const groupCreator = groupCreatorsDetails[group.createdBy];
+      const groupCreator = groupCreatorsDetails.find((user) => user.id === group.createdBy);
```

Comment:
Why is this being changed here?

Reply:
Because of [this](https://github.com/Real-Dev-Squad/website-backend/pull/1630/files#diff-9a5062203df8184c1d64d655f2dcd16eebeea484b834b0f461771b463037ceefL668-R668) . I changed the user response format from object to array.